.github/workflows/ci.yaml

```diff
@@ -111,7 +106,27 @@ jobs:
           - php-version: '7.4'
             composer-options: '--prefer-stable'
             symfony-version: '5.3.*'
+          - php-version: '7.4'
+            composer-options: '--prefer-stable'
+            symfony-version: '5.4.*'
           - php-version: '8.0'
+            composer-options: '--prefer-stable'
+            symfony-version: '5.3.*'
```

Comment:
[Support for Symfony 5.3 ends soon](https://symfony.com/releases/5.3). I think it would be ok to remove it from the test matrix at least for newer PHP versions.

Reply:
Ok, I removed the entries for 8.0 & 8.1 👍